src/index.mjs

```diff
@@ -66,6 +74,7 @@ export const createUploadLink = (
           .then(response => {
             if (!response.ok)
               throw new Error(`${response.status} (${response.statusText})`)
+            operation.setContext({ response })
```

Comment:
This should come before the response check above. Otherwise if the response is not ok, the response will not be attached to the context like in `apollo-link-http`.